modules/run_clinical_release.nf

```diff
@@ -0,0 +1,28 @@
+process run_clinical_release {
+
+   container 'sagebionetworks/genie-bpc-pipeline-clinical-release'
+   secret 'SYNAPSE_AUTH_TOKEN'
+   debug true
+
+   input:
+   val previous
+   val cohort
+   val production
+
+   output:
+   stdout
+
+   script:
+   if (production) {
+      """
+      cd /usr/local/src/myscripts/
+      Rscript create_release_files.R -c $cohort -v -s
+      """
+   }
+   else {
+      """
+      cd /usr/local/src/myscripts/
+      Rscript create_release_files.R -c $cohort -v
```

Comment:
I also noticed there were other release related scripts in the `release` directory, any reason those are not being run?

Reply:
Needs comment from @Chelsea-Na to see what other scripts should be added or nextflow processes